extensions/amp-list/amp-list.md

```diff
@@ -0,0 +1,79 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-list"></a> `amp-list`
+
+The `amp-list` fetches the dynamic content from a CORS JSON endpoint and renders it
+using a supplied template.
+
+#### Usage
+
+The `amp-list` defines data source using the following attributes:
+
+- `src` defines a CORS URL. The URL's protocol must be HTTPS.
+- `credentials` defines a `credentials` option as specified by the
+[Fetch API](https://fetch.spec.whatwg.org/). To send credentials, pass the
+value of "include".
+
+The request URL is allowed to contain the following substitutions:
+
+- CANONICAL_URL
+- TODO
+
+The response must be a JSON object that contains an array property "items":
+```json
+{
+  "items": []
+}
+```
+
+The template can be specified using either of the following two ways:
+
+- `template` attribute that references an ID of an existing `template` element.
+- `template` element nested directly inside of this `amp-list` element.
+
+For more details on templates see [AMP HTML Templates](../../spec/amp-html-templates.md).
+
+An example:
+```html
+<amp-list src="https://data.com/articles.json?ref=CANONICAL_URL">
+  <template type="amp-mustache">
+    <div>
```

Comment:
So, there are some issues with `<li>` inside of `<template>` on some browser :( Our polyfill is not that good. Instead, I think we should do role=list/listitem